dll/win32/oleaut32/oleaut.c

```diff
@@ -660,6 +660,7 @@ ULONG WINAPI OaBuildVersion(void)
     case 0x00000005:  /* W2K */
 		return MAKELONG(0xffff, 40);
     case 0x00000105:  /* WinXP */
+    case 0x00000205:  /* Win2K3 */
```

Comment:
It's not guarded anymore. Did you sent a Merge Request to Wine GitLab?

Reply:
Nope, I'm new to contribute files from wine, what should I do now? guarded or sent to wine?

Reply:
Send it to Wine. At least open a bug report and attach your Patch as well as your test program. After that you can create a merge request which refers to your Bug report.
Maybe our Wine ambassador Fabian Maurer can help to get it in?

Reply:
I proposed it as a Wine ticket here : https://bugs.winehq.org/show_bug.cgi?id=54257
Just update with your final implementation.

Reply:
Committed, `#ifdef __REACTOS__` guarded now